src/main/java/org/gridsuite/merge/orchestrator/server/dto/MergeConfig.java

```diff
@@ -24,5 +25,7 @@
 
```

Comment:
I think we don't use this class anymore : to be deleted

Reply:
class deleted